routing_model/index.html

```diff
@@ -132,8 +141,59 @@ <h2>Node as a member of a section</h2>
 </div>
 
   <h1>Destination section</h1>
-  Receiving a node that is being relocated.
+  <p>As a member of a section, our section will sometimes receive a node that is being relocated. These  diagrams are from the point of view of one of the nodes in the section, doing its part to handle the node that is trying to relocate to this section.
+  </p>
   <h2>Processing a candidate trying to join our section</h2>
+  <p>This flow represents what we do when a candidate contacts us to join our section.<br>
+  The process starts as we receive an ExpectCandidate RPC from this node.<br>
```

Comment:
> This flow represents what we do when a candidate contacts us to join our section.
> The process starts as we receive an ExpectCandidate RPC from this node.

The flow is from a section contacting us with a candidate. ExpectCandidate is from the section.